msteams-platform/messaging-extensions/create-api-message-extension.md

```diff
@@ -158,22 +164,62 @@ To build am API-based message extension using Teams Toolkit for Visual Studio Co
         |`appPackage/responseTemplates/repair.json`     |  A generated Adaptive Card that used to render API response.       |
         |`repairsData.json`    |  The data source for the repair API.       |
 
+   1. Based on the options selected in **step a**, follow these steps:
```

Comment:
👆The above table also need to be updated.

Reply:
Ho @huimiu ,

I've updated the table respectively for Visual Studio and Visual Studio Code. 

thanks,
Yash